jsonnet/kube-prometheus/addons/aws-vpc-cni.libsonnet

```diff
@@ -77,9 +77,13 @@
       apiVersion: 'monitoring.coreos.com/v1',
       kind: 'PrometheusRule',
       metadata: {
-        labels: $.prometheus._config.commonLabels + $.prometheus._config.mixin.ruleLabels,
+        labels: {
+          'app.kubernetes.io/name': 'prometheus-vpc-cni-rules',
+          'app.kubernetes.io/component': 'prometheus',
+          'app.kubernetes.io/part-of': 'kube-prometheus',
+        },
```

Comment:
For this part, I couldn't find a good replacement for this as we no longer have access to `$.prometheus` in this scope. Please let me know if you have any suggestions on how this can be improved so we don't have to hard-coded these values here.